File: ARCADE/Intro/Level 11_1: isDigit.cs
    using System;
     
    public class Test
    {
    	public static bool isDigit(char symbol) {
     
    	    return ('0' <= symbol && symbol <= '9');
     
    	}
    	public static void Main()
    	{
    		char symbol = '9';
    		Console.Write(isDigit(symbol));
    	}
    }

File: ARCADE/Intro/Level 11_1: isDigit.pas
program isDigit;

function isDigit(symbol: char): boolean;
begin
    if((48 <= ord(symbol)) and (ord(symbol) <= 57)) then isDigit := true
    else isDigit := false;
end;

var symbol : char;
begin
	symbol := '0';
	writeln(isDigit(symbol));
	symbol := '-';
	writeln(isDigit(symbol));
end.
